src/EventTable.tsx

```diff
@@ -71,30 +71,35 @@ interface StyledTableProps {
 
 export default function StyledTable({ events, setModalData, theme, options }: StyledTableProps): ReactElement {
   const Style = styled.div`
-    table{
-      width:100%
-    }
-    
     .fixed_header{
       table-layout: fixed;
+      width: 100%;
     }
   
     .fixed_header tbody{
-      display:block;
+      display: block;
       width: 100%;
-      overflow: auto;
+      overflow: overlay;
       height: ${options.height - 21}px;
     }
   
-    .fixed_header thead tr {
-      display: block;
+    .header_row{
+      display: flex;
+      color: #33A2E5;
+      width: 100%;
     }
-  
-  
-    .fixed_header th, .fixed_header td {
+
+    .fixed_header th{
+      width: 14.2857143%;
+      text-align: left;
+      padding: 6px;
+      border: 1px solid #9a9a9a;
+    }
+     
+    .fixed_header td{
+      width: 14.2857143%;
       text-align: left;
-      padding-left: 5px;
-      width: 300px;
+      padding: 6px;
       border: 1px solid #9a9a9a;
```

Comment:
apply to right border only, change color to `rgb(32, 34, 38)` add a bottom border to `tr`'s below